main.js

```diff
@@ -0,0 +1,15 @@
+const p = document.querySelector("#clickcount");
+const storage = window.localStorage;
+
+let i = storage.getItem("clickcount");
```

Comment:
`i` it's a bad name for a variable. Rename on `count`, `counter` or something like that.